common-ui/src/main/res/color/button_destructive_container_selector.xml

```diff
@@ -14,10 +15,7 @@
   ~ limitations under the License.
   -->
 
-<shape xmlns:android="http://schemas.android.com/apk/res/android">
-    <stroke
-        android:width="1.5dp"
-        android:color="@color/cornflowerBlue" />
-
-    <corners android:radius="4dp" />
-</shape>
\ No newline at end of file
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="?attr/daxColorDestructive" android:state_enabled="true"/>
+    <item android:color="?attr/daxColorContainerDisabled"/>
```

Comment:
Don't we need the attribute `android:state_enabled="false"` for this item?

Reply:
No, this is a `backgroundTint`. I've just slightly modified the default [MaterialButton backgroundTint](https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/button/res/color/mtrl_btn_bg_color_selector.xml)